packages/core/src/interactions/packet/acknowledgement.ts

```diff
@@ -43,8 +41,9 @@ const metric_receivedFailedAcks = create_counter(
 )
 const metric_sentAcks = create_counter('core_counter_sent_acks', 'Number of sent message acknowledgements')
 
-const metric_winningTickets = create_counter('core_counter_winning_tickets', 'Number of winning tickets')
-const metric_losingTickets = create_counter('core_counter_losing_tickets', 'Number of losing tickets')
+const metric_ackedTickets = create_counter('core_counter_acked_tickets', 'Number of acknowledged tickets')
+
+const PREIMAGE_PLACE_HOLDER = new Hash(new Uint8Array({ length: Hash.SIZE }).fill(1))
```

Comment:
why don't use a zero-filled place-holder or fill with `.fill(0xff)`? Filling with `0x01` can easily be misinterpreted as something meaningful

Reply:
Agree. I'd change it to fill with 0xff